tests/data/test1933

```diff
@@ -32,7 +32,7 @@ Content-Length: 0
 <server>
 http
 </server>
-# this relies on the debug feature which allow to set the time
+# this relies on the debug feature which allows one to set the time
```

Comment:
"allows tests" perhaps?

Reply:
Done in 6f7208a53f3df4a2cf19ffc9bdd743aa5f51e9ed